Core/UniversalAppLinkRecovery.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  UniversalAppLinkRecovery.swift
```

Comment:
This sounds like it specifically responds to universal app link recovery - but it really reloads any errors that happen while backgrounding. Can we rename? Maybe BackgoundErrorRecovery?

Reply:
Actually, that's a good point.  We don't want to do anything if the user backgrounds the app after an error under normal circumstances.  Need to think about this.